packages/utils/find-root.ts

```diff
@@ -0,0 +1,8 @@
+import * as findup from "findup-sync";
+import * as path from "path";
+
+export function findProjectRoot(curDir: string): string {
```

Comment:
`curDir` is not used, you can remove it

Reply:
Fixed, I thought in future I can make this extendible, but removing it for now.